src/System.Security.Cryptography.Xml/src/System.Security.Cryptography.Xml.csproj

```diff
@@ -91,6 +91,9 @@
     <Compile Include="System\Security\Cryptography\Xml\RSAPKCS1SHA512SignatureDescription.cs" />
     <Compile Include="System\Security\Cryptography\Xml\RSAPKCS1SHA384SignatureDescription.cs" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
This project shouldn't reference System.Security.Permissions

Reply:
+1, the fix has to go into the ref project, not the src project.